src/components/SecurityHeader.js

```diff
@@ -48,30 +46,23 @@ class SecurityHeader extends React.Component {
 		}
 
 		return (
-			<TouchableItem onPress={() => this.props.navigation.navigate('Security')}>
-				<View
-					style={{
-						alignItems: 'center',
-						flexDirection: 'row',
-						paddingRight: 14
-					}}
-				>
-					<Icon style={styles.headerSecureIcon} name="security" />
-					<IdentitiesSwitch />
-				</View>
-			</TouchableItem>
+			<View
+				style={{
+					alignItems: 'center',
+					flexDirection: 'row',
+					paddingRight: 14
+				}}
+			>
+				<IdentitiesSwitch />
+				<ButtonIcon
+					onPress={() => this.props.navigation.navigate('Security')}
+					iconName="security"
+					iconColor={colors.bg_alert}
+				/>
+				<ButtonIcon iconType="ionicon" iconName="md-more" />
```

Comment:
if here use `onPress={()=> {}}`, then maybe we can keep the `props: onPress` as `isRequired`, so later we wont't forget to add any onPress button for the ButtonIcon, I suppose it will always come with an `onPress` function?

Reply:
sure, done in [205fbdc](https://github.com/paritytech/parity-signer/pull/431/commits/205fbdcef81c5df61941a34702bf10e2ccbb5bf5)